docs/configuration.md

```diff
@@ -26,13 +26,69 @@ variables.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_REDIS_USERNAME
+
+- Redis ACL user name
```

Comment:
Do we need `ACL` here? When I read it I'm starting to wonder, what is the difference between a username and an ACL username :angel:

Reply:
I'm gonna remove it.

Reply:
:thinking: